### PR TITLE
NX Monorepo

### PR DESCRIPTION
**Pull request by Sabo2k**

### CODE REVIEW

**Comment by ivandukov** [comment]
whats wrong with tanstack query?

**Comment by ivandukov** [comment]
@Sabo2k should we push the nx cache? i don't think so?

File: .nx/cache/19.0.2-nx.linux-x64-gnu.node
Comment (ivandukov) [inline .nx/cache/19.0.2-nx.linux-x64-gnu.node:1]: ?

Comment (Sabo2k) [reply .nx/cache/19.0.2-nx.linux-x64-gnu.node:1]: Yeah you're right. Thanks!

**Pull request merged**